### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Monsterhunter0815**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -30,5 +30,36 @@
...
+		"haptics",
+		"lern",
+		"lerportal",
+		"pcss",
+		"pnpx",
+		"preinstall",
+		"schulsolutions",
+		"stylelint",
+		"tailwindcss"
+	]
```
Comment (3x071c) [inline .vscode/settings.json]: Can you remove this? I'm used to saving the dictionary words under the user profile instead...

File: src/components/App.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ export default function App(): JSX.Element {
 			<IonReactRouter>
 				<IonSplitPane contentId='main'>
 					<Menu mount={mount} />
+					<ion-icon name='alarm-outline' />
```
Comment (3x071c) [inline src/components/App.tsx:17]: I think this should be `IonIcon`

Comment (3x071c) [reply src/components/App.tsx:17]: Also, why is there an icon between the menu and the body anyway?

File: src/components/Card.tsx [hunk 3]
```diff
@@ -1,9 +1,13 @@
+/* eslint-disable react/jsx-no-comment-textnodes */
```
Comment (3x071c) [inline src/components/Card.tsx:1]: This rule exists for a reason, see below...

File: src/components/Card.tsx [hunk 4]
```diff
@@ -1,9 +1,13 @@
...
 import clsx from 'clsx';
 import type { Props } from '$util/types';
 
+//* Card takes in contents and returns the full Card */
 export default function Card({ children, className, ...props }: Props): JSX.Element {
 	return (
+		//* Main div section containing props */
```
Comment (3x071c) [inline src/components/Card.tsx]: This comment is not valid, it will be rendered! Could wrap it in curly braces.

File: src/components/Feed.tsx [hunk 5]
```diff
@@ -14,6 +15,7 @@ import { homeItems as homeItemsOrigin } from '$mock/index';
 import { notificationsOutline } from 'ionicons/icons';
 import FeedCard from '$components/FeedCard';
 import Notifications from '$components/Notifications';
+import React from 'react';
```
Comment (3x071c) [inline src/components/Feed.tsx:18]: No need for the import I believe?

File: src/components/Card.tsx (hunk 4 above)
Comment (Monsterhunter0815) [reply src/components/Card.tsx]: it is, wrapping it in curly braces throws an error

**Pull request closed without merging**